client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -168,6 +175,8 @@ const CheckoutThankYou = React.createClass( {
 				const purchase = find( purchases, isGoogleApps );
 
 				page( upgradesPaths.domainManagementEmail( this.props.selectedSite.slug, purchase.meta ) );
+			} else {
+				page( `/stats/insights/${ this.props.selectedSite.slug }` );
```

Comment:
Instead of `if-else`, what about `if-return`? I think it'd make it look nicer and we can get rid of multiple elses that act the same.